packages/patternfly-4/react-core/src/components/Dropdown/Toggle.js

```diff
@@ -51,8 +51,9 @@ class DropdownToggle extends Component {
   };
 
   onEscPress = event => {
+    const { parentRef } = this.props;
     const keyCode = event.keyCode || event.which;
-    if (keyCode === 27) {
+    if (keyCode === 27 && parentRef && parentRef.contains(event.target)) {
```

Comment:
There is a constant for the escape key defined in the internal/constants.js can we use that here?